app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt

```diff
@@ -34,7 +34,7 @@ class ProfileProgressViewModel @Inject constructor(
   private val topicListController: TopicListController,
   private val logger: ConsoleLogger,
   @StoryHtmlParserEntityType private val entityType: String
-) : ObservableViewModel() {
```

Comment:
Does it just not need to be observable?

Reply:
@BenHenning Though it is assigned to Akshay, I am actually not able to understand what do you mean by this comment?

Reply:
Ah. I was confused about why this was removed, but it makes sense now.